src/coralogix.js

```diff
@@ -81,21 +81,9 @@ export class CoralogixLogger {
         body: JSON.stringify(payload),
       }));
       if (!resp.ok) {
-        let msg = `Failed to send logs with status ${resp.status}: ${await resp.text()}`;
-        if (resp.status === 400) {
-          msg = `${msg}\nlogEntries: ${JSON.stringify(logEntries, 0, 2)}`;
-        }
-        return new Response(msg, { status: resp.status });
+        throw Error(`Failed to send logs with status ${resp.status}: ${await resp.text()}`);
```

Comment:
this service doesn't have an API gateway, so we generally no longer send `Response`s bad throw up to AWS.